package.json

```diff
@@ -25,32 +25,24 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:unit": "wct --config-file ./wct.local.conf.json",
-    "test:e2e": "npm run test:server:start && npm run test:e2e:chrome && npm run test:e2e:firefox",
-    "test:e2e:grid": "npm run test:server:start && npm run test:e2e:chrome:grid && npm run test:e2e:firefox:grid",
-    "test:grid": "wct --config-file ./wct.docker.conf.json && npm run test:e2e:grid",
-    "posttest:e2e": "pkill -f http-server",
-    "test:e2e:chrome": "mocha --chrome",
-    "test:e2e:chrome:grid": "mocha --chrome --grid",
-    "test:e2e:firefox": "mocha --firefox",
-    "test:e2e:firefox:grid": "mocha --firefox --grid",
-    "test:e2e:opera": "mocha --opera",
-    "test:e2e:safari": "mocha --safari",
-    "test:server:start": "http-server &",
-    "release": "np",
+    "test": "npm explore vl-ui-util -- npm run test:component",
+    "test:grid": "npm explore vl-ui-util -- npm run test:component:grid",
+    "test:unit": "npm explore vl-ui-util -- npm run test:component:unit",
+    "test:e2e": "npm explore vl-ui-util -- npm run test:component:e2e",
+    "test:e2e:grid": "npm explore vl-ui-util -- npm run test:component:e2e:grid:all",
```

Comment:
waarom hier nog de :all ?

Reply:
kunt ge dan wel nog met "makkelijke" commando's lokaal testen zonder docker? dat vind ik persoonlijk een vereiste ...

Reply:
> waarom hier nog de :all ?

Ik ben ook niet helemaal mee met het verschil tussen script met of zonder `:all` vermoedelijk nog wat residu?

Reply:
> waarom hier nog de :all ?

Om in de docker-compose te gebruiken.

> kunt ge dan wel nog met "makkelijke" commando's lokaal testen zonder docker? dat vind ik persoonlijk een vereiste ...

Ja, `npm run test` runt zowel unit als E2E. `npm run test:e2e --browser=chrome` enkel de E2E testen in chrome